### ISSUE TITLE
Autoloader Problem with PHPExcel (v. 3.9)

### ISSUE THREAD

**Issue by Sogl** [koala-framework/koala-framework#357]
Hi!

PHPExcel don't work in 3.9:
```
<br />
<b>Fatal error</b>:  Class 'PHPExcel' not found in <b>C:\OpenServer\domains\aviashelf\controllers\FormEx
.php</b> on line <b>47</b><br />
<html>
    <head>
        <title>500 Internal Server Error</title>
    </head>
    <body>
        <h1>Error</h1>

                    Errortype:<br />
            <pre></pre>

            <br />

            Message:<br />
            <pre>exception 'ErrorException' with message 'Class 'PHPExcel' not found' in C:\OpenServer
\domains\aviashelf\controllers\FormEx.php:47
Stack trace:
#0 [internal function]: Kwf_Setup::shutDown()
#1 {main}</pre>
                                    <p id="exception" style="display:none">czoxOTg6ImV4Y2VwdGlvbiAnR
XJyb3JFeGNlcHRpb24nIHdpdGggbWVzc2FnZSAnQ2xhc3MgJ1BIUEV4Y2VsJyBub3QgZm91bmQnIGluIEM6XE9wZW5TZXJ2ZXJcZ
G9tYWluc1xhdmlhc2hlbGZcY29udHJvbGxlcnNcRm9ybUV4LnBocDo0NwpTdGFjayB0cmFjZToKIzAgW2ludGVybmFsIGZ1bmN0aW9uXTogS3dmX1NldHVwOjpzaHV0RG93bigpCiMxIHttYWlufSI7
</p>            </body>
</html>
```

My composer:
```
{
    "require": {
        "koala-framework/koala-framework": "3.9.x-dev",
        "phpoffice/phpexcel": "1.8.0"
    },
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

Line causes an error:
```
$xls = new PHPExcel();
```

In 3.8 all works fine.

**Comment by nsams** [comment]
got probably broken by 027c182ad643b8e64647608b1fee02acc102d619

**Comment by Ben-Ho** [comment]
content of composer autoload_namespace.php does already include _ in key: (except PHPExcel...)

return array(
    'Zend_' => array($vendorDir . '/koala-framework/zendframework1/library'),
    'Vkwf_' => array($vendorDir . '/vivid-planet/vkwf'),
    'Vkwc_' => array($vendorDir . '/vivid-planet/vkwf'),
    'Twig_' => array($vendorDir . '/twig/twig/lib'),
    'Sepia' => array($vendorDir . '/sepia/po-parser/src'),
    'Psr\\Log\\' => array($vendorDir . '/psr/log'),
    'PHPExcel' => array($vendorDir . '/phpoffice/phpexcel/Classes'),
    'Kwf_SourceMaps_' => array($vendorDir . '/koala-framework/sourcemaps'),
    'Kwf_Owlcarousel_' => array($vendorDir . '/koala-framework/kwf-owlcarousel'),
    'Kwf_' => array($vendorDir . '/koala-framework/koala-framework'),
    'Kwf\\Lingohub\\' => array($vendorDir . '/koala-framework/kwf-lingohub'),
    'Kwf\\FileWatcher' => array($vendorDir . '/koala-framework/file-watcher/src'),
    'Kwf\\Deploy\\' => array($vendorDir . '/koala-framework/kwf-deploy'),
    'Kwc_Parallax_' => array($vendorDir . '/koala-framework/kwc-parallax'),
    'Kwc_' => array($vendorDir . '/koala-framework/koala-framework'),
);

**Comment by nsams** [comment]
fixed in 3.9 with #369
